configure

```diff
@@ -52,7 +52,7 @@ pie=1
 arch=`uname -s`
 cpu=`uname -m`
 
-GLIB_MIN_VERSION=2.14
+GLIB_MIN_VERSION=2.16
```

Comment:
I was just thinking about this.. How about having a policy or so, and always allow this to be bumped up up to the glib version currently in Debian's oldest LTS release? (2.24 ATM)

Not to bump it up now without using its features, but if there's a good reason to do so, then do it.